.gitignore

```diff
@@ -177,3 +177,4 @@ dmypy.json
 
 # Mac stuff
 *.DS_Store
+data/regionmapping/region_df.csv
```

Comment:
I wonder if we really want to add this to the gitignore? 
Shouldn't we track it when we want to update it?